apisix/plugins/jwt-auth.lua

```diff
@@ -295,6 +299,10 @@ local function find_consumer(conf, ctx)
         return nil, nil, "failed to verify jwt"
     end
 
+    if conf.store_in_ctx then
+        ctx.jwt_obj = jwt_obj
```

Comment:
Awesome, maybe it would be better called `jwt_payload`, or `jwt_auth_payload`? 

Use `payload`, (and specifically prefix it with the jwt-auth plugin name, as some other plugins such as openid-connect also use JWT, which may cause conflicts